### PR TITLE
Pro 3750 upload check your answers

### PR DESCRIPTION
**Pull request by harjeetkalsi**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PRO-3750

### CODE REVIEW

File: app/steps/ui/summary/index.js [hunk 1]
```diff
@@ -87,7 +87,9 @@ class Summary extends Step {
             .replace('{deceasedName}', deceasedName ? deceasedName : content.DeceasedAlias.theDeceased);
         ctx.deceasedMarriedQuestion = (hasCodicils ? content.DeceasedMarried.questionWithCodicil : content.DeceasedMarried.question)
             .replace('{deceasedName}', deceasedName);
-        ctx.uploadedDocuments = formdata.documents.uploads.map(doc => doc.filename);
+        if (formdata.documents && formdata.documents.uploads) {
+            ctx.uploadedDocuments = formdata.documents.uploads.map(doc => doc.filename);
+        }
```
[the remark this answers is not in the document]
Comment (harjeetkalsi) [reply app/steps/ui/summary/index.js]: I think this has been done in testSummary.js unit test where I have written 2 tests under getContextData() which cover this logic ?

File: test/component/summary/testDocumentUploadSection.js [hunk 2]
```diff
@@ -16,13 +16,13 @@ describe('summary-documentupload-section', () => {
...
 
-    describe.only('Verify Content, Errors and Redirection', () => {
+    describe('Verify Content, Errors and Redirection', () => {
         it('test correct content loaded on document upload section of the summary page , when no data is entered', (done) => {
             const playbackData = {
             };
             testWrapper.testDataPlayback(done, playbackData);
         });
-
+PRO
```
[the remark this answers is not in the document]
Comment (harjeetkalsi) [reply test/component/summary/testDocumentUploadSection.js]: removed!

File: test/component/summary/testDocumentUploadSection.js [hunk 3]
```diff
@@ -1,9 +1,8 @@
 'use strict';
 
-const requireDir = require('require-directory');
 const TestWrapper = require('test/util/TestWrapper');
 const documentuploadData = require('test/data/documentupload');
-const documentuploadContent = requireDir(module, '../../../app/resources/en/translation/documentupload');
+const documentuploadContent = require('../../../app/resources/en/translation/documentupload.json');
```
[the remark this answers is not in the document]
Comment (harjeetkalsi) [reply test/component/summary/testDocumentUploadSection.js]: done

File: app/steps/ui/summary/template.html [hunk 4]
```diff
@@ -21,6 +21,11 @@ <h2 class="heading-medium">
         {% include "ui/summary/includes/will.njk" %}
     </dl>
 
+    <h2 class="heading-medium">{{content.Summary.uploadedDocumentsHeading}}</h2>
+    <dl class="check-your-answers check-your-answers--long">
+        {% include "ui/summary/includes/documentupload.njk" %}
+    </dl>
```
[the remark this answers is not in the document]
Comment (harjeetkalsi) [reply app/steps/ui/summary/template.html]: yes I have now added a ctx.isDocumentUploadToggleEnabled in summary index.js and call it in the template.

**Comment by shepda** [comment]
Closed because it has been replaced with https://github.com/hmcts/probate-frontend/pull/470

**Pull request closed without merging**